### PR TITLE
BMS? More like RTD

### PR DESCRIPTION
**Pull request by mjmagee991**
Updated lots of documentation: RTD, README, and inline docs

### CODE REVIEW

**Comment by mjmagee991** [comment]
[BMS Documentation Update](https://trello.com/c/mi8Z0qGL/164-bms-documentation-update)

File: include/BQSettingStorage.hpp [hunk 1]
```diff
@@ -123,23 +130,24 @@ class BQSettingsStorage {
     BMS::DEV::BQ76952::Status transferSetting(bool& isComplete);
 
     /**
-     * Checks to see if the settings are stored and can be used. This includes
-     * checking to see if there are settings stored, and also checks to make
-     * sure settings are not actively being written to.
+     * Check if the settings are stored and can be used
+     *
+     * This includes checking to see if there are settings stored, and also
+     * checks to make sure settings are not actively being written to.
```
Comment (ActuallyTaylor) [inline include/BQSettingStorage.hpp:136]: It would be best if you had an @return because your other Boolean functions have them.

**Review by ActuallyTaylor — CHANGES_REQUESTED** [review]

**Review by ActuallyTaylor — APPROVED** [review]
10/10

**Comment by ol2764RIT** [comment]
great docs

**Review by ol2764RIT — APPROVED** [review]
good

**Pull request merged**